# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-before-rust-1.86.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-134.0.6998.45/build/rust/std/BUILD.gn.vanilla	2025-03-06 08:40:20.613053176 +0100
+++ chromium-134.0.6998.45/build/rust/std/BUILD.gn	2025-03-06 08:40:30.195077806 +0100
@@ -89,7 +89,7 @@
     # These are no longer present in the Windows toolchain.
     stdlib_files += [
       "addr2line",
-      "adler2",
+      "adler",
       "gimli",
       "libc",
       "memchr",
